setup.cfg

```diff
@@ -37,6 +37,8 @@ install_requires =
     kornia>=0.5.4
     matplotlib
     numpy
+    # omegaconf 2.1+ required for to_object method
+    omegaconf>=2.1
```

Comment:
Technically omegaconf isn't used in torchgeo proper, only in train.py/benchmark.py and the unit tests, but it seemed weird to keep a `train` list of optional deps with only one dep...